packages/ui-kit/src/components/Switch/Switch.tsx

```diff
@@ -4,15 +4,12 @@ import { Switch as HeadlessSwitch } from "@headlessui/react";
 
 import styles from "./Switch.module.css";
 import labelStyles from "../Label/Label.module.css";
+import { ClassNames } from "../../globalTypes";
```

Comment:
Maybe we could add some `@` alias so these imports are not relative. In some latter pr of course

Reply:
Done.